pkg/webhook/internal/cert/generator/selfsigned.go

```diff
@@ -30,24 +32,42 @@ func ServiceToCommonName(serviceNamespace, serviceName string) string {
 
 // SelfSignedCertGenerator implements the certGenerator interface.
 // It provisions self-signed certificates.
-type SelfSignedCertGenerator struct{}
+type SelfSignedCertGenerator struct {
+	CAKey  []byte
+	CACert []byte
```

Comment:
why are they public ? (if they are public then probably we don't need the `SetCA` method ?